carplanner_msgs/OdometryArray odom_arr
bool loop


